src/providers/entries/GateEntry.ts

```diff
@@ -8,10 +8,14 @@ import {
 import { getStaticResult } from '../../lib/configUtils';
 import { StatsigConfig } from '../../state/ProjectsState';
 import { APIConfigEntry } from './APIConfigEntry';
+import { ConfigGroupEntry } from './ConfigGroupEntry';
 
 export class GateEntry extends APIConfigEntry {
-  constructor(public readonly data: StatsigConfig) {
-    super(data);
+  constructor(
+    public readonly data: StatsigConfig,
+    readonly parent: ConfigGroupEntry,
+  ) {
+    super(data, parent);
     if (!data.data.enabled) {
       this.iconPath = CONFIG_DISABLED_ICON;
```

Comment:
so this has changed since it was written - its now possible to be disabled and return true, or disabled and return false.

Reply:
Yeah we should revisit this. I'm not sure if we should keep this indicator based on enabled/disabled or 100% pass/0%pass. Regardless, for purposes of this diff, we use the default value for replacement.